tools/quantize/transform.py

```diff
@@ -58,3 +61,71 @@ def __init__(self, ctype):
 
     def __call__(self, img):
         return cv.cvtColor(img, self.ctype)
+
+class ImgAlign:
```

Comment:
Should name it `HandAlign` since it only works with hand models. If conditions `if model == "mp_handpose":` can be remove then.